regolith/helpers/a_proposalhelper.py

```diff
@@ -28,70 +23,57 @@ def subparser(subpi):
     subpi.add_argument("name", help="A short but unique name for the proposal",
                        default=None
                        )
-    subpi.add_argument("-amt", "--amount", help="value of award",
+    subpi.add_argument("amount", help="value of award",
                        default=None
                        )
-    subpi.add_argument("-a", "--authors",
-                       help="Other investigator names"
+    subpi.add_argument("authors", help="Other investigator names"
                        )
-    subpi.add_argument("--begin_day", nargs="+",
-                       help="The start day for the proposed grant in format YYYY-MM-DD."
+    subpi.add_argument("currency", help="Currency in which amount is specified."
+                                        " Typically $ or USD"
                        )
-    subpi.add_argument("--begin_month", nargs="+",
-                       help="The start month for the proposed grant in format YYYY-MM-DD."
+    subpi.add_argument("due_date", help="The due date for the proposal in format YYYY-MM-DD."
                        )
-    subpi.add_argument("--begin_year", nargs="+",
-                       help="The start year for the proposed grant in format YYYY-MM-DD."
+    subpi.add_argument("duration", help="Duration of proposal in years"
                        )
-    subpi.add_argument("--end_date", nargs="+",
-                       help="The end date for the proposed grant in format YYYY-MM-DD."
+    subpi.add_argument("submitted_date", help="Day on which the proposal is submitted "
```

Comment:
please make optional. In general I won't know this when I create the proposal so will rarely, if ever, use it.  In fact, maybe delete it from the arg list?

Reply:
done